README.md

```diff
@@ -1,2 +1,19 @@
 # import_service_terraform
 Terraform for import service
+
+
+TODOs:
+
++ create project terra-importservice-env
++ create import service SA
++ give import SA pubsub admin and token creator permissions in terra-importservice-env
++ give terra-importservice-env GAE SA token creator permissions on import service SA
++ database
++ pubsub topic and subscription
++ create bucket for translated imports
++ firewall around GAE
+
+
+manual (unless terraform can run arbitrary sh):
```

Comment:
It can, an example of a script that was recently incorporated into TF is here:
https://github.com/broadinstitute/terraform-terra/pull/125

Reply:
The tricky bit is making sure it's idempotent and fails in a recoverable manner.